src/operation-poller.ts

```diff
@@ -0,0 +1,82 @@
+import { Queue } from "./throttler/queue";
+import * as api from "./api";
+import * as FirebaseError from "./error";
+
+export interface OperationPollerOptions {
+  pollerName?: string;
+  apiOrigin: string;
+  apiVersion: "v1" | "v1beta1";
```

Comment:
Can you add a comment pointing to the resource where these versions came from? Probably the Cloud docs somewhere...

Reply:
Since this is a generic utility that could be used against any API, I think I'd prefer just making `apiVersion` a `string`.

Reply:
@mbleigh sounds good!
@bkendall, FYI you can checkout the proposal ([go/firebase-cli-project-management](https://goto.google.com/firebase-cli-project-management)) 's Public REST API endpoints section. I'm polling from both Cloud (v1) and Firebase (v1beta1) endpoints

Reply:
Done

Reply:
(Now that this is a string it matters less, but:)

Design proposals are lost context once this code is submitted. Having an in-line comment saying "here's where you can check what the available versions are" is infinitely better (IMO).